client/galaxy/scripts/mvc/tools.js

```diff
@@ -388,7 +389,7 @@ var ToolSearch = Backbone.Model.extend({
         this.timer = setTimeout(function () {
             // log the search to analytics
             ga( 'send', 'pageview', galaxy_config.root + '?q=' + q );
-            $.get(self.attributes.search_url, { query: q }, function (data) {
+            $.get( tool_search_url, { q: q }, function ( data ) {
```

Comment:
It makes *more* sense to hack together a url in the middle of the search method?